packages/app-desktop/gui/MenuBar.tsx

```diff
@@ -1140,7 +1143,7 @@ function MenuBar(props: Props): any {
 
 
 const mapStateToProps = (state: AppState): Partial<Props> => {
-	const whenClauseContext = stateToWhenClauseContext(state);
+	const whenClauseContext = stateToWhenClauseContext(state, { windowId: state.windowId });
```

Comment:
This change was required for "Focus viewer" to be correctly enabled/disabled in secondary windows.